sui/src/bin/validator.rs

```diff
@@ -100,7 +100,11 @@ async fn main() -> Result<(), anyhow::Error> {
     );
 
     let consensus_committee = network_config.make_narwhal_committee();
-    let consensus_parameters = ConsensusParameters::default();
+    let consensus_parameters = ConsensusParameters {
+        max_header_delay: 5_000,
+        max_batch_delay: 5_000,
+        ..ConsensusParameters::default()
+    };
```

Comment:
Maybe not for this PR, but let's consider reading those from a file. Eventually we might want to get from env variables injected to the Docker image, but let's discuss about that.

Reply:
There is an issue for this https://github.com/MystenLabs/sui/issues/1553